demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -274,23 +275,20 @@ def from_path(
                 logger.error(f"Could not parse content from {str(path)}")
                 return None
         try:
+            content_item.MARKETPLACE_MIN_VERSION = "0.0.0"
```

Comment:
why? please add a comment

Reply:
Just moved it from a different part of the code.